pkg/aws/ami.py

```diff
@@ -17,86 +17,98 @@ async def start_ami_creation_worker(bucket_name, image_name, project, disk_conta
    secret_key = Project.objects(name=project).allow_filtering()[0]['secret_key']
    region = Project.objects(name=project).allow_filtering()[0]['location']
    import_task_id = ''
+   role_id = "vmimport"
```

Comment:
can this name be created dynamically per project?

Reply:
The reason why I didn't make it dynamic is because the we are adding no restricted permissions for vmimport. We don't specify the resources but give full access so the role and permissions are always going to be the same.